components/AppBar/README.md

```diff
@@ -60,9 +60,9 @@ The Material Design top app bar displays information and actions relating to the
   - [Enabling top layout guide adjustment](#enabling-top-layout-guide-adjustment)
   - [Enabling inferred top safe area insets](#enabling-inferred-top-safe-area-insets)
 - [Extensions](#extensions)
-  - [Color Theming](#color-theming)
-  - [Typography Theming](#typography-theming)
-  - [Theming Extensions (Beta)](#theming-extensions-(beta))
+  - [Theming](#theming)
+  - [Color Theming (To be deleted)](#color-theming-(to-be-deleted))
+  - [Typography Theming (To be deleted)](#typography-theming-(to-be-deleted))
```

Comment:
Please fix these links. This will be broken on the website.

https://github.com/material-components/material-components-ios/issues/7068